### PR TITLE
Fix codecov tests

### PR DESCRIPTION
**Pull request by alejoe91**
Thansk for the hint @h-mayorquin 

Tests passing: https://github.com/SpikeInterface/spikeinterface/actions/runs/12157608869

### CODE REVIEW

File: .github/workflows/all-tests.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
             echo "$file was changed"
           done
 
-      - name: Set testing environment  # This decides which tests are run and whether to install especial dependencies
+      - name: Set testing environment  # This decides which tests are run and whether to install special dependencies
```
Comment (h-mayorquin) [inline .github/workflows/all-tests.yml:50]: My Spanish speaking self came out yelling on this one.

**Review by h-mayorquin — APPROVED** [review]
Great!

**Comment by h-mayorquin** [comment]
The failing tests seem to be IBL : /

**Review by zm711 — APPROVED** [review]
This also looks good to me :) It'll be nice to have the green box again !

**Comment by alejoe91** [comment]
> The failing tests seem to be IBL : /

Pinned one-api...

**Pull request merged**